testplan/testing/multitest/driver/tcp/server.py

```diff
@@ -152,7 +153,14 @@ def receive(self, size=None, conn_idx=None, timeout=30):
     def starting(self):
         """Starts the TCP server."""
         super(TCPServer, self).starting()
-        self._server = Server(host=self.cfg.host, port=self.cfg.port)
+        self._server = Server(
+            host=self.cfg.host,
+            port=(
+                expand(self.cfg.port, self.context, int)
```

Comment:
the expanded port might be a string as well, need to transform that

Reply:
The third param will be used as the constructor similar to the tcp_client commend